### DISCUSSION TITLE
sent e-mails tab

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#108] [General]
Hi Franky
I use the template you give in your documentation (hook) to send an e-mail when a new event is scheduled. How can I make this e-mail also appear in the sent e-mails tab so that I can follow the good reading of the recipients ?

Frédéric

**Comment by liedekef** [comment]
Change
`      eme_send_mail($subject,$body, $user->user_email, $user->display_name, $contact_email, $contact_name);`
to
`      eme_queue_mail($subject,$body, $contact_email, $contact_name, $user->user_email, $user->display_name);`

(the sequence of arguments is different, historical ...)

**Reply by f14m07** [reply]
I have a fatal error because eme_queue_mail function seems to need two other arguments

**Reply by liedekef** [reply]
indeed, the next version will fix that. For now:
`eme_queue_mail($subject,$body, $contact_email, $contact_name, $user->user_email, $user->display_name, $contact_email, $contact_name);`

(the sequence is: subject, body, from-info, to-info, replyto-info)

**Comment by f14m07** [comment]
1/ is it not rather (the sequence is: subject, body, **to-info**, from-info, replyto-info)
On my local site it works in this order
2/ how will the next version fix this? because I don't want my snippet to break the site on the next update

**Reply by liedekef** [reply]
No, check the code if you want to check, I already said the sequence is different. Which you should do if want to use functions from a plugin anyway ...
The next version won't break this, but as always: check the changelog upon upgrade.

**Reply by liedekef** [reply]
btw: the to-info is 2 arguments (email, name), so are from-info and replyto-info (I used "info" to indicate general purpose). See eme-mail.php for the function definition.

**Reply by f14m07** [reply]
Thanks Franky I made a mistake in the order of the parameters
